introduction.md

```diff
@@ -2,4 +2,8 @@ Hi, I'm Haipt
 I'm .Net Developer at DVG
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?